plugins/src/web/config.ron

```diff
@@ -131,6 +131,14 @@
             matches: ["wiki"],
             queries: [(name: "Wikipedia", query: "wikipedia.org/w/index.php?search=")]
         ),
+        (
+            matches: ["www"],
+            queries: [(
+                name: "Open Website",
+                query: "http://",
```

Comment:
Perhaps we should default to https?

Reply:
I was really split on this. I could go either way. I think there are a very small handful of URLs I visit that are not https, but it doesn't feel right that they should make all the other URLs insecure by default.